contact.html

```diff
@@ -0,0 +1,127 @@
+      crossorigin="anonymous"
+    />
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-ygbV9kiqUc6oa4msXn9868pTtWMgiQaeYH7/t7LECLbyPA2x65Kgf80OJFdroafW"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="./css/main.css" />
+    <script
+      src="https://kit.fontawesome.com/d835f4ba90.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Spectral:ital@1&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Spectral:ital,wght@0,500;1,400&display=swap"
+      rel="stylesheet"
+    />
+
+    <title>contact</title>
+  </head>
+
+  <body>
+    <div class="row" style="background-color: #e2f7cd">
+      <div class="container">
+        <div class="col text-center m-5">
+          <p class="italic">Contact Me</p>
+          <h1>I'm a title. ​Click here to edit me.</h1>
+          <p>
+            I'm a paragraph. Click here to add your own text and edit me. I’m a
+            great place for you to tell a story and let your users know a little
+            more about you.
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="row" style="background-color: #e2f7cd">
+      <div class="col">
+        <div class="container">
+          <form class="row g-3 pt-4 pb-4">
+            <div class="col-md-6 m-0">
+              <label for="name" class="form-label"></label>
+              <input
+                type="text"
+                class="form-control"
+                id="inputEmail4"
+                placeholder="Name"
+              />
+            </div>
+            <div class="col-md-6 m-0">
+              <label for="email" class="form-label"></label>
+              <input
+                type="email"
+                class="form-control"
+                id="inputPassword4"
+                placeholder="Email"
+              />
+            </div>
+            <div class="col-12 m-0">
+              <label for="subject" class="form-label"></label>
+              <input
+                type="text"
+                class="form-control"
+                id="inputAddress"
+                placeholder="Subject"
+              />
+            </div>
+            <div class="col-12 m-0">
+              <label for="message" class="form-label"></label>
+              <textarea
+                class="form-control"
+                id="exampleFormControlTextarea1"
+                rows="5"
+                placeholder="Message"
+              ></textarea>
+            </div>
+            <div class="d-grid gap-2">
+              <button class="btn btn-primary" type="button">Send</button>
+            </div>
+          </form>
```

Comment:
Try to write all your CSS styles inside your CSS file instead of using the HTML `style` attribute.